### DISCUSSION TITLE
use aws mysql jdbc driver HikariCP 3.4.5 only write failover time too long than HikariCP 2.7.5

### DISCUSSION THREAD

**Discussion by zixubingfeng** [awslabs/aws-mysql-jdbc#207] [Q&A]
use HikariCP 3.4.5 only write failover time need 10 minutes ,but use HikariCP 2.7.5 only wrote failover time only 10 seconds,help me to check this question

**Comment by hsuamz** [comment]
There has been improvements in this area with our latest releases. We'll be closing this discussion. Please feel free to re-open this discussion if there are further questions. Thank you.
